definitions/ext-dell_compellent/summary_metrics.yml

```diff
@@ -0,0 +1,7 @@
+queueDepthPercent:
```

Comment:
any reason to have a different summary than a golden?

We are working towards removing summary metrics so this will evntually be removed in favor of golden metrics

Reply:
summary metrics are shown on the explorer list view. Golden metrics are shown on the navigator view when click on an entity - What will be the difference in this regard when summary metrics are removed? @naxhh

Reply:
Golden metrics are the main important metrics of an entity and are displayed in several places.

I understand those will be the ones displayed also in the explorer list view at some point